community/blog/2021-12 2021, the year that Arcus matured/README.md

```diff
@@ -0,0 +1,173 @@
+
+Our goal for the next year, is to make the message handling available for Azure Functions so it would look like this:
+
+```csharp
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Abstractions.ServiceBus;
+using Arcus.Messaging.Abstractions.ServiceBus.MessageHandling;
+using Azure.Messaging.ServiceBus;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+public class OrderProcessingFunction
+{
+    private readonly IAzureServiceBusMessageRouter _messageRouter;
+    private readonly string _jobId;
+
+    public OrderProcessingFunction(IAzureServiceBusMessageRouter messageRouter)
+    {
+        _messageRouter = messageRouter;
+        _jobId = Guid.NewGuid().ToString();
+    }
+
+    [FunctionName("order-processing")]
+    public async Task Run(
+        [ServiceBusTrigger("docker-az-func-queue", Connection = "ARCUS_SERVICEBUS_CONNECTIONSTRING")] ServiceBusReceivedMessage message,
+        ILogger log,
+        CancellationToken cancellationToken)
+    {
+        log.LogInformation($"C# ServiceBus queue trigger function processed message: {message.MessageId}");
+
+        AzureServiceBusMessageContext context = message.GetMessageContext(_jobId);
+        MessageCorrelationInfo correlationInfo = message.GetCorrelationInfo();
+
+        await _messageRouter.RouteMessageAsync(message, context, correlationInfo, cancellationToken);
+    }
+}
+```
+
+[Another blog post](https://www.codit.eu/blog/taking-messaging-to-the-next-level-with-arcus-messaging-v1-0/) has been written on this major change.
+See our [features docs](https://messaging.arcus-azure.net/) for more information on our available functionality.
+
+## Observability
+The most important 2021 changes in the [observability repository](https://github.com/arcus-azure/arcus.observability) are all related to our upcoming [service-to-service correlation](https://github.com/arcus-azure/arcus.observability/issues?q=is%3Aissue+is%3Aopen+label%3Aservice-to-service-correlation) functionality. This feature would allow us to link requests across components in Azure Application Insights. Naturally, this is quite a big and complex adaptation so we choose to split the functionality in separate tasks.
+
+The changes that were made this year on this, are:
+* Capability to add operation parent ID to correlation information
+* Capability to add dependency name to dependency telemetry
+
+Besides those changes, we are working hard to make this library production-ready. It's a popular and very important library to have mature telemetry in your project.
```

Comment:
'we are working hard to make this library production-ready'; shouldn't we say something like 'make it even more production worthy'?

Reply:
Yes, you're right. 👍